src/status_im/ui/screens/wallet/send/subs.cljs

```diff
@@ -66,7 +66,8 @@
   :<- [:balance]
   (fn [[{:keys [amount to] :as transaction} balance]]
     (assoc transaction :sufficient-funds? (or (nil? amount)
-                                              (money/sufficient-funds? amount balance)))))
+                                              ;; TODO(jeluard) Modify to consider tokens
+                                              (money/sufficient-funds? amount (get balance :ETH))))))
```

Comment:
Maybe `(:eth balance)`?

Reply:
https://github.com/bbatsov/clojure-style-guide#lisp-case and https://github.com/bbatsov/clojure-style-guide#keywords-as-fn-to-get-map-values

Reply:
Right but this will be changed soon to rely on a variable symbol.

Reply:
"Soon", the mythical date that gets further and further pushed into the future ;)

Explicitly approving since it is minor stylistic thing and PR fixes a regression in develop.